flac.md

```diff
@@ -232,11 +232,9 @@ The MD5 signature is made by performing an MD5 transformation on the samples of
 
 ## Padding
 
-The padding metadata block allows for an arbitrary amount of padding. The contents of a padding block have no meaning. This block is useful when it is known that metadata will be edited after encoding; the user can instruct the encoder to reserve a padding block of sufficient size so that when metadata is added, it will simply overwrite the padding (which is relatively quick) instead of having to insert it into the right place in the existing file (which would normally require rewriting the entire file).
+The padding metadata block allows for an arbitrary amount of padding. This block is useful when it is known that metadata will be edited after encoding; the user can instruct the encoder to reserve a padding block of sufficient size so that when metadata is added, it will simply overwrite the padding (which is relatively quick) instead of having to insert it into the right place in the existing file (which would normally require rewriting the entire file).
```

Comment:
Add "There MAY be one or more padding metadata blocks per FLAC stream." for clarifying the answer to another question in https://github.com/xiph/flac/issues/544 and mirroring a sentence in other parts of the spec e.g. "There MUST be no more than one streaminfo metadata block per FLAC stream".